docs/Parameters/Mobilogram.rst

```diff
@@ -14,10 +14,10 @@ Parameters
 Example Usage
 -------------
 
+To create a mobilogram plot, you can use the following example scripts:
```

Comment:
Remove or add a statement like this to all plot classes (PeakMap, Chromatogram, Spectrum) for consistency.